### PR TITLE
Added examples for NLB, ALB ingress and service account

### PR DESCRIPTION
**Pull request by simeonzhekofff**
- Added examples for deploy the chart with:
- Service: Load Balancer type
- Ingress with ALB
- Service Account

### CODE REVIEW

File: examples/aws/ingress/values.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This example shows how to deploy and expose GraphDB with AWS Load Balancer Controller Ingress.
+
+configuration:
+  externalUrl: graphdb-example-dns-name.com # Change this to your ALB DNS name or Route53 if you use it.
+
+service:
+  enabled: true
+  type: NodePort
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:8]: You don't need this for the Ingress scenario

File: examples/aws/ingress/values.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  port: 7200
+
+ingress:
+  enabled: true
+  annotations:
+    ingressClassName: "alb"
+    alb.ingress.kubernetes.io/load-balancer-name: graphdb-ingress
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}]'
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:18]: I wonder if 443 example would be helpful.

File: examples/aws/ingress/values.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    alb.ingress.kubernetes.io/inbound-cidrs: "0.0.0.0/0"
+    alb.ingress.kubernetes.io/backend-protocol: HTTP
+  labels:
+    app: graphdb-0
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:22]: What's the purpose of this label?

File: examples/aws/ingress/values.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  rules:
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:23]: There is NO such configuration in values.yaml

File: examples/aws/ingress/values.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+service:
+  enabled: true
+  type: NodePort
+  port: 7200
+
+ingress:
+  enabled: true
+  annotations:
+    ingressClassName: "alb"
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:14]: We have `ingress.className` for this

File: examples/aws/ingress/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+* [Exposing kubernetes applications via ingress](https://aws.amazon.com/blogs/containers/exposing-kubernetes-applications-part-1-service-and-ingress-resources/)
+
+## Example
+* [values.yaml](values.yaml) - Example of how to deploy and expose GraphDB with Ingress.
+
+## Note
+After you deploy and the ingress is created please change the externalUrl value to the DNS name of the ALB or Route53.
+
+
+
```
Comment (mihailradkov) [inline examples/aws/ingress/README.md:19]: :unamused:

File: examples/aws/ingress/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Overview 
+
+This document describes the steps on how to configure the GraphDB helm chart to use ingress on AWS EKS
```
Comment (mihailradkov) [inline examples/aws/ingress/README.md:3]: Helm begins with a capital letter. 
I would capitalize Ingress as well

Comment (mihailradkov) [inline examples/aws/ingress/README.md:3]: Missing dot at the end of the sentence.

File: examples/aws/ingress/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+## Prerequisites
+
+* EKS Cluster: Ensure you have an EKS cluster up and running.
+* [Installing AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/deploy/installation/)
+* [Ingress Setup](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html)
+* [Exposing kubernetes applications via ingress](https://aws.amazon.com/blogs/containers/exposing-kubernetes-applications-part-1-service-and-ingress-resources/)
+
+## Example
```
Comment (mihailradkov) [inline examples/aws/ingress/README.md:12]: Always leave an empty line around headings and paragraphs.

File: examples/aws/lb-example/README.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# EKS Load Balancer Type Deployment
+
+# Pre-requisites 
+
+* EKS Cluster: Ensure you have an EKS cluster up and running.
+* [Installing AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/deploy/installation/)
+* [AWS Documentation - Route TCP and UDP traffic with Network Load Balancers](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
+
+This folder contains examples of using GraphDB with the AWS Network Load Balancer.
```
Comment (mihailradkov) [inline examples/aws/lb-example/README.md:9]: Move this before the prereqs

File: examples/aws/lb-example/README.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# EKS Load Balancer Type Deployment
+
+# Pre-requisites
```
Comment (mihailradkov) [inline examples/aws/lb-example/README.md:3]: This should be a level 2 heading

File: examples/aws/lb-example/values.yaml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+ loadBalancerClass: service.k8s.aws/nlb
+ annotations:
+  service.beta.kubernetes.io/aws-load-balancer-name: "graphdb-lb"
+  service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: "ip"
+  service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+  service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"  # This ensures the load balancer is internet-facing
+  service.beta.kubernetes.io/load-balancer-source-ranges: "0.0.0.0/0"
+  service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "" # SSL Certificate that will be used
+  service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "" # Ports to be used for the SSL
+ metadata:
```
Comment (mihailradkov) [inline examples/aws/lb-example/values.yaml:15]: There is NO such configuration in `values.yaml`

File: examples/aws/ingress/values.yaml [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+  rules:
+    - http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: graphdb
+                port:
+                  number: 7200
```
Comment (mihailradkov) [inline examples/aws/ingress/values.yaml:32]: Missing empty line at the end.

File: examples/aws/service-account/values.yaml [hunk 13]
```diff
@@ -0,0 +1,5 @@
+serviceAccount:
+  create: true
+  name: graphdb
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::<AWS_ACCOUNT_ID>:role/eks-service-account-role # Example ARN Role, replace with your actual IAM Role ARN
```
Comment (mihailradkov) [inline examples/aws/service-account/values.yaml:5]: Missing empty line at the end

File: examples/aws/ingress/values.yaml (hunk 1 above)
Comment (simeonzhekofff) [reply examples/aws/ingress/values.yaml:8]: Forgot to remove it.

File: examples/aws/ingress/values.yaml (hunk 2 above)
Comment (simeonzhekofff) [reply examples/aws/ingress/values.yaml:18]: I will add one.

File: examples/aws/ingress/values.yaml (hunk 3 above)
Comment (simeonzhekofff) [reply examples/aws/ingress/values.yaml:22]: Will be removed, I tried something

File: examples/aws/ingress/README.md (hunk 8 above)
Comment (mihailradkov) [reply examples/aws/ingress/README.md:13]: There is one more that should be addressed but another time.

**Review by mihailradkov — APPROVED** [review]

File: examples/aws/ingress/README.md (hunk 8 above)
Comment (simeonzhekofff) [reply examples/aws/ingress/README.md:13]: Fixed

**Pull request merged**